podaac/subsetter/subset.py

```diff
@@ -1110,9 +1117,35 @@ def get_coordinate_variable_names(dataset, lat_var_names=None, lon_var_names=Non
                 dataset, dataset[lat_var_name]
             ) for lat_var_name in lat_var_names
         ]
+        time_var_names.append(compute_utc_name(dataset))
+        time_var_names = list(dict.fromkeys([x for x in time_var_names if x is not None]))  # remove Nones and any duplicates
+
     return lat_var_names, lon_var_names, time_var_names
 
 
+def convert_to_datetime(dataset, time_vars):
+    """
+    converts the time variable to datetime if xarray decode times
```

Comment:
did you mean "if xarray cannot decode times"?

Reply:
Yes